### PR TITLE
[DDW-1121] Update delegation copy

### PR DESCRIPTION
**Pull request by daniloprates**
This PR applies text updates and countdown for the following screens:

## Delegation center next epoch notification
dce
dcj
image

## Delegation wizard "TADA" screen
dwe
dwj
image

## Undelegation dialog final step
ue
uj
image

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1576118824364700)

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by ManusMcCole** [comment]
The time does not get translated to Japonese for Storybook.Checking Daedalus now

**Comment by ManusMcCole** [comment]
In the app in the delegation success screen I see an English "and" that didnt get translated and a "Cardano" missing thats in the images above 
JaponeseDelegated Success

**Comment by ManusMcCole** [comment]
Delegate success screen has an unstranslated "and" and also the undelegated screen 
JaponeseDelegated Success

**Comment by ManusMcCole** [comment]
Undelegation Success screen also has an untranslated "and"
UndelegatedScreenand

**Comment by tomothespian** [comment]
@daniloprates Untranslated part for next epoch date is here `humanizeDuration`, in both dialogs. 
@ManusMcCole storybook is not a problem because "en-us" is HC to story.

File: source/renderer/app/api/network/types.js [hunk 1]
```diff
@@ -40,4 +46,8 @@ export type NetworkInfoResponse = {
     epoch_number: number,
     epoch_start: string,
   },
+  future_epoch: {
```
Comment (nikolaglumac) [inline source/renderer/app/api/network/types.js]: @daniloprates we don't need this.

File: source/renderer/app/api/utils/patchAdaApi.js [hunk 2]
```diff
@@ -28,8 +28,13 @@ export default (api: AdaApi) => {
...
       Logger.debug('AdaApi::getNetworkInfo (PATCHED) success', { networkInfo });
 
-      /* eslint-disable-next-line camelcase */
-      const { sync_progress, node_tip, network_tip, next_epoch } = networkInfo;
+      const {
+        sync_progress, // eslint-disable-line camelcase
+        node_tip, // eslint-disable-line camelcase
+        network_tip, // eslint-disable-line camelcase
+        next_epoch, // eslint-disable-line camelcase
+        future_epoch, // eslint-disable-line camelcase
```
Comment (nikolaglumac) [inline source/renderer/app/api/utils/patchAdaApi.js]: @daniloprates please revert this part as we don't get `future_epoch` from the API.

File: source/renderer/app/api/utils/patchAdaApi.js [hunk 3]
```diff
@@ -50,6 +55,10 @@ export default (api: AdaApi) => {
           epochNumber: get(next_epoch, 'epoch_number', 0),
           epochStart: get(next_epoch, 'epoch_start', ''),
         },
+        futureEpoch: {
```
Comment (nikolaglumac) [inline source/renderer/app/api/utils/patchAdaApi.js]: @daniloprates please refactor to match: https://github.com/input-output-hk/daedalus/pull/1762/files#diff-6aa9e91ae277a5d21b8e1bb0b03ffafbR1167

File: source/renderer/app/api/network/types.js (hunk 1 above)
Comment (daniloprates) [reply source/renderer/app/api/network/types.js]: ✅

File: source/renderer/app/api/utils/patchAdaApi.js (hunk 2 above)
Comment (daniloprates) [reply source/renderer/app/api/utils/patchAdaApi.js]: ✅

File: source/renderer/app/api/utils/patchAdaApi.js (hunk 3 above)
Comment (daniloprates) [reply source/renderer/app/api/utils/patchAdaApi.js]: ✅

**Pull request merged**